src/main/tray/reactive-tray-menu-items/convert-to-electron-menu-template.injectable.ts

```diff
@@ -0,0 +1,63 @@
+/**
+ * Copyright (c) OpenLens Authors. All rights reserved.
+ * Licensed under MIT License. See LICENSE in root directory for more information.
+ */
+import { getInjectable } from "@ogre-tools/injectable";
+import loggerInjectable from "../../../common/logger.injectable";
+import type { TrayMenuItem } from "../tray-menu-item/tray-menu-item-injection-token";
+
+const convertToElectronMenuTemplateInjectable = getInjectable({
```

Comment:
Hmm. This could be just a function (not injectable at all). Basically it's just a mapper function. Logger is dependency which can be removed since the implementations should handle it separately. (and they already do, so it's just left-over from refactorings)

Reply:
I would partially agree that code is harder to read now, I guess this is one of drawbacks from injectables. Another problem I can mention is no documentation about DI tools we use which slows the pace a lot and leads to write code without full understanding. Plus code navigation is hard.

On the other hand, we got better test deploying + separation of concerns which is good.

Reply:
This is good conversation. Remember that not all the benefits of complying to **Dependency Inversion Principle** or the **Dependency Inversion Container** pattern are apparent right away. For one, after a coder gains familiarity, e.g. navigation actually gets easier since dependencies are more likely to do one thing well, and describe that in its name with credibility. This should mean less need to visit the code. Also from that, we will have stuff like composability, clear abstractions, decoupling and cohesion. Cool stuff right guys? :)